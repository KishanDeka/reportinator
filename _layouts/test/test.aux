\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\zref@newlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {I}Objective}{1}{section.1}}
\newlabel{sec:intro}{{I}{1}{Objective}{section.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {II}Theory}{1}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {II-A}Electron Spin Resonance:}{1}{subsection.2.1}}
\newlabel{key}{{4}{1}{Electron Spin Resonance:}{equation.2.4}{}}
\newlabel{key}{{5}{1}{Electron Spin Resonance:}{equation.2.5}{}}
\newlabel{key}{{6}{1}{Electron Spin Resonance:}{equation.2.6}{}}
\newlabel{key}{{7}{1}{Electron Spin Resonance:}{equation.2.7}{}}
\newlabel{eq}{{8}{1}{Electron Spin Resonance:}{equation.2.8}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {II-B}ExpEyes:}{1}{subsection.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {II-B.1}Electromagnetic Induction:}{1}{subsubsection.2.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {II-B.2}Induced Voltage in a photodiode:}{2}{subsubsection.2.2.2}}
\@writefile{toc}{\contentsline {section}{\numberline {III}Data and Observations:}{2}{section.3}}
\@writefile{lot}{\contentsline {table}{\numberline {I}{\ignorespaces Electron spin resonance data\relax }}{2}{table.caption.1}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{tab:my-esr}{{I}{2}{Electron spin resonance data\relax }{table.caption.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {IV}Graphs:}{2}{section.4}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces ESR Plot\relax }}{2}{figure.caption.2}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Plotted Induced EMF\relax }}{2}{figure.caption.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {3}{\ignorespaces Time-Voltage plot for the photodiode\relax }}{2}{figure.caption.4}}
\@writefile{toc}{\contentsline {section}{\numberline {V}Results and Conclusion:}{2}{section.5}}
\@writefile{toc}{\contentsline {section}{\numberline {VI}Precautions:}{2}{section.6}}
\@writefile{toc}{\contentsline {section}{\numberline {VII}References}{2}{section.7}}
