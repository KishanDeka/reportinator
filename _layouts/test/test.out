\BOOKMARK [1][-]{section.1}{Objective}{}% 1
\BOOKMARK [1][-]{section.2}{Theory}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{Electron Spin Resonance:}{section.2}% 3
\BOOKMARK [2][-]{subsection.2.2}{ExpEyes:}{section.2}% 4
\BOOKMARK [3][-]{subsubsection.2.2.1}{Electromagnetic Induction:}{subsection.2.2}% 5
\BOOKMARK [3][-]{subsubsection.2.2.2}{Induced Voltage in a photodiode:}{subsection.2.2}% 6
\BOOKMARK [1][-]{section.3}{Data and Observations:}{}% 7
\BOOKMARK [1][-]{section.4}{Graphs:}{}% 8
\BOOKMARK [1][-]{section.5}{Results and Conclusion:}{}% 9
\BOOKMARK [1][-]{section.6}{Precautions:}{}% 10
\BOOKMARK [1][-]{section.7}{References}{}% 11
